roles/docker-options/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+docker_log_max_size: 256m
+docker_log_max_file: 4
```

Comment:
Needs EOL char

Reply:
fixed, but that is not shown here in GitHub